src/test/ui/async-await/suggest-missing-await.rs

```diff
@@ -0,0 +1,31 @@
+// edition:2018
+
+#![feature(async_await)]
+
+fn take_u32(x: u32) {}
+
+async fn make_u32() -> u32 {
+    22
+}
+
+async fn foo1() {
+    let x = make_u32();
+    take_u32(x) //~ ERROR mismatched types [E0308]
```

Comment:
There should also be `HELP` and `SUGGESTION` annotations for the cases with the suggestion.